Dockerfile

```diff
@@ -1,5 +1,7 @@
 FROM python:3-slim-buster
 
+RUN useradd --create-home --home-dir /app appuser
```

Comment:
Should the user be `fig` with an application directory of `/fig`? Seems like we can be more opinionated here.